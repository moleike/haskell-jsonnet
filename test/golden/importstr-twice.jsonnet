local x = importstr "empty.jsonnet";
local y = importstr "empty.jsonnet";
x + y
