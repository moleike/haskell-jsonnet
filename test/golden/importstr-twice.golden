"{ foo :: \"bar\" }\n{ foo :: \"bar\" }\n"